soh/soh/Enhancements/speechsynthesizer/DarwinSpeechSynthesizer.h

```diff
@@ -5,8 +5,7 @@
 //  Created by David Chavez on 22.11.22.
 //
 
-#ifndef SOHDarwinSpeechSynthesizer_h
-#define SOHDarwinSpeechSynthesizer_h
+#pragma once
```

Comment:
nitpick: `#pragma once` is non-standard. I'd prefer to keep the include guards that you changed in this PR (they can be updated to follow better practices of naming like `FILENAME_H` though)

Reply:
we use `#pragma once` throughout codebase, a consistent policy should be made